components/formatter/IngressTarget.vue

```diff
@@ -63,7 +63,7 @@ export default {
         const paths = rule?.http?.paths || [];
 
         return paths.map(path => ({
-          url:             `${ rule.host }${ path.path }`,
+          url:             `${ rule.host }${ path.path || '' }`,
```

Comment:
It looks like the same thing will happen with `rule.host` if it's undefined here